### PR TITLE
update to beta

### PR DESCRIPTION
**Pull request by rmatambo8**
Updated to beta new tags

tested:
- package.json is up to date
- had problems with `env` files both for `yarn start` and `yarn build`  so  I added dotenv package and updated the path inside `gatsby-config`
- there were some performance errors in the console but nothing breaking. (see suggested edit)
- ran the build, start and serve command
- when I the serve command and the content is not served but the other commands work

I'd love to see if someone else is getting the same issue (serve not serving content after building) locally.

I can give some environment variables to my sample shopify store for testing

### CODE REVIEW

**Comment by rmatambo8** [comment]
my suggestion was to memoize the creation of the current product list for the search, its pretty expensive and since we are also debouncing we may be able to get a minor but easy gain. Since i had problems with serve I thought this could be a different discussion

```
  // If we're using the default filters, use the products from the Gatsby data layer.
  // Otherwise, use the data from search.
  const isDefault = !data
  const productList = (isDefault ? products.edges : data?.products?.edges) ?? []

  //... changes to:

  // If we're using the default filters, use the products from the Gatsby data layer.
  // Otherwise, use the data from search.
  const isDefault = !data
  const productList = React.useMemo(() => {
    return (isDefault ? products.edges : data?.products?.edges) ?? []
  }, [data, products, isDefault])
```

File: src/pages/search.jsx [hunk 1]
```diff
@@ -112,7 +112,9 @@ function SearchPage({
   // If we're using the default filters, use the products from the Gatsby data layer.
   // Otherwise, use the data from search.
   const isDefault = !data
-  const productList = (isDefault ? products.edges : data?.products?.edges) ?? []
+  const productList = React.useMemo(() => {
```
Comment (jxnblk) [inline src/pages/search.jsx:115]: Let's keep unrelated changes out of scope here. Was this causing issues with one of the updated packages?

Comment (rmatambo8) [reply src/pages/search.jsx:115]: no, this was just a performance improvement - the console had the suggestion to memoize this and it was a pretty easy change so i did it

Comment (rmatambo8) [reply src/pages/search.jsx:115]: reverting now

**Comment by rmatambo8** [comment]
@jxnblk I'm gonna merge since the issue has been addressed. Any objections?

**Review by aghreed — APPROVED** [review]
Code wise this LGTM. I need to look into setting up a shopify account though to give this a full run through locally.
